src/controller.js

```diff
@@ -0,0 +1,130 @@
+(function exportController() {
+
+  const shipDiv = document.getElementById('ship');
+
+  class Controller {
+    constructor(ship) {
+      this.ship = ship;
+      document.getElementById('sailbutton').addEventListener('click', () => this.setSail());
+      this.renderDisplay();
+    }
+
+    initialiseSea() {
+      let counter = 1;
+      function changeBackground() {
+        const viewport = document.querySelector('#viewport');
```

Comment:
better have `const` declared at the top of an execution block, in this case, in line 13